### ISSUE TITLE
New version AnimateJS

### ISSUE THREAD

**Issue by ghost** [matthieua/WOW#346]
Hi! A new version of animate.js has been released, I think it's time for you to update too!

**Comment by andyatroam** [comment]
Yes, just tested it, it doesn't work with the new version of the [animate.css](https://animate.style/). The name convention of the CSS has been changed like this `animate__animated animate__heartBeat`

**Comment by veraLX** [comment]
Hello，我是林昕，我已收到来信~

**Comment by Plaristote** [comment]
It woud be nice to update the defaults, but it still works using the `animateClass` option:

```js
import wowjs from "wowjs";

const wow = new wowjs.WOW({
  animateClass: "animate__animated"
});

document.addEventListener("DOMContentLoaded", function() { wow.init(); });
```
